assets/scss/modules/_epaye-statements.scss

```diff
@@ -2,8 +2,103 @@
   border-bottom-width: 0;
 }
 
+#delegation-banner {
+  background-color:$info-bg-color; 
+  border-left: 3px solid $govuk-blue;
+  margin: 10px 0 1em; 
+  padding: 0.6em 1em 0.5em; 
+  max-width: 900px;
+  float: none;
+  clear: both;
+  z-index: 999;
+  
+  a.client-list {
+    margin-left: 19px;
+  }
+}
+.paye-header {
+  h1 {
+    padding: 10px 0 5px 0;
+    margin-bottom: 10px;      
+  }
+}
+
+a.paye-back-link {
```

Comment:
this selector's overqualified. does it need to be?

also, it can extend [the already existing back link](https://github.com/hmrc/assets-frontend/blob/b9cdbc7235884ad696f02983e6ba064c1963869d/assets/scss/modules/_navigation.scss#L145) as a modifier: i.e. `.back-link--black`